concepticondata/conceptlists/Hantgan-2022-351.tsv

```diff
@@ -0,0 +1,352 @@
+Hantgan-2022-351-79	79	DYE	tissu noirâtre teinté d'indigo	157	DYE
+Hantgan-2022-351-80	80	EAR	oreille	1247	EAR
+Hantgan-2022-351-81	81	EARRING	bague ou boucle d’oreille -cpd: see [kàmb-ù]-[kòrb-ò] , hà.è-[kòrb-ò] , cè:-[kòrb-ò] .	770	EARRING
+Hantgan-2022-351-82	82	EASY	ê. facile, pas cher	686	EASY
+Hantgan-2022-351-83	83	EAT	manger	1336	EAT
+Hantgan-2022-351-84	84	EGG	oeuf	744	EGG
+Hantgan-2022-351-85	85	EIGHT	huit	1705	EIGHT
+Hantgan-2022-351-86	86	ELBOW	coude	981	ELBOW
+Hantgan-2022-351-87	87	ELEPHANT	éléphant	1290	ELEPHANT
+Hantgan-2022-351-88	88	EMPTY	(récipient) vide	1624	EMPTY
+Hantgan-2022-351-89	89	ENTER	entrer	749	ENTER
+Hantgan-2022-351-90	90	EYE	œil	1248	EYE
+Hantgan-2022-351-91	91	FACE	visage	1560	FACE
+Hantgan-2022-351-92	92	FALL	tomber	1280	FALL
+Hantgan-2022-351-93	93	FAR	éloigner, écarter	1406	FAR
+Hantgan-2022-351-94	94	FAST	rapide	1631	FAST
+Hantgan-2022-351-95	95	FAT (OBESE)	gros	1279	FAT (OBESE)
+Hantgan-2022-351-96	96	FAT (ORGANIC SUBSTANCE)	graisse	323	FAT (ORGANIC SUBSTANCE)
+Hantgan-2022-351-97	97	FATHER	père	1217	FATHER
+Hantgan-2022-351-98	98	FEAR (BE AFRAID)	avoir peur	1419	FEAR (BE AFRAID)
+Hantgan-2022-351-99	99	FEAR (FRIGHT)	peur	781	FEAR (FRIGHT)
+Hantgan-2022-351-100	100	FIGHT	lutte	1423	FIGHT
+Hantgan-2022-351-101	101	FIND	obtenir, gagner	719	FIND
+Hantgan-2022-351-102	102	FINGER	doigt	1303	FINGER
+Hantgan-2022-351-103	103	FIRE	feu	221	FIRE
+Hantgan-2022-351-104	104	FIREWOOD	bois (de chauffage)	10	FIREWOOD
+Hantgan-2022-351-105	105	FISH	poisson	227	FISH
+Hantgan-2022-351-106	106	FIVE	cinq	493	FIVE
+Hantgan-2022-351-107	107	FLOWER	fleur	239	FLOWER
+Hantgan-2022-351-108	108	FLY (INSECT)	mouche	1504	FLY (INSECT)
+Hantgan-2022-351-109	109	FLY (MOVE THROUGH AIR)	(oiseau etc.) voler	1441	FLY (MOVE THROUGH AIR)
+Hantgan-2022-351-110	110	FOAM	bulles, écume, mousse (de savon, de thé) [verb kù:fà ].	1768	FOAM
+Hantgan-2022-351-111	111	FOLD	plier (papier, vêtement)	1470	FOLD
+Hantgan-2022-351-112	112	FOOD	aliment, nourriture	256	FOOD
+Hantgan-2022-351-113	113	FOOT OR LEG	pied	2098	FOOT OR LEG
+Hantgan-2022-351-114	114	FOREHEAD	front	123	FOREHEAD
+Hantgan-2022-351-115	115	FORGET	oublier	1523	FORGET
+Hantgan-2022-351-116	116	FORGIVE	excuser, pardonner (qqch) -ex: .ò né [.jéy .=´ .´yà:fà] they said they (had) forgiven it (e.g. debt)	1528	FORGIVE
+Hantgan-2022-351-117	117	FOUR	quatre	1500	FOUR
+Hantgan-2022-351-118	118	FRIEND	ami	1325	FRIEND
+Hantgan-2022-351-119	119	FULL	plein	1429	FULL
+Hantgan-2022-351-120	120	GET	obtenir	694	GET
+Hantgan-2022-351-121	121	GIRL	fille	1646	GIRL
+Hantgan-2022-351-122	122	GIVE	donner	1447	GIVE
+Hantgan-2022-351-123	123	GIVE BIRTH	(femelle) accoucher (bébé)	1195	GIVE BIRTH
+Hantgan-2022-351-124	124	GO	aller	695	GO
+Hantgan-2022-351-125	125	GO DOWN (DESCEND)	descendance	2014	GO DOWN (DESCEND)
+Hantgan-2022-351-126	126	GO OUT	descendre	2014	GO DOWN (DESCEND)
+Hantgan-2022-351-127	127	GO UP (ASCEND)	monter	705	GO UP (ASCEND)
+Hantgan-2022-351-128	128	GoːUT	quitter, venir de (lieu)r	1107	GO OUT
```

Comment:
Is the English gloss "Go:UT" correct?

Reply:
It is incorrect. I'd leave it rather unmapped, quitter in French is like "leave", but I find it hard to match it really with "LEAVE".